src/sites/ptt/bot.ts

```diff
@@ -203,15 +218,15 @@ class Bot extends EventEmitter {
     return ret;
   }
 
-  async logout() {
+  async logout(): Promise<boolean> {
     if (!this.state.login) return;
     await this.send(`G${key.Enter}Y${key.Enter.repeat(2)}`);
     this._state.login = false;
     this.emit('stateChange', this.state);
     return true;
   }
 
-  async _checkLogin(kick) {
+  async _checkLogin(kick: boolean): Promise<any> {
```

Comment:
`private` is able to be used in Typescript

Reply:
Done